src/handlers/tracker.ts

```diff
@@ -253,7 +258,7 @@ export class TrackerHandler implements IDisposable {
 
   protected findInReadOnlyEditors(_: string, source: string) {
     let editors: CodeEditor.IEditor[] = [];
-    this.readOnlyEditorTracker.forEach(widget => {
+    this.readOnlyEditorTracker?.forEach(widget => {
```

Comment:
Is `?` needed? `readOnlyEditorTracker` is instantiated in the constructor and should not be `null` or `undefined`.

Reply:
i had problem during unit test that's why i put that there. I will try without ?

Reply:
What problem?

Reply:
![issue](https://user-images.githubusercontent.com/34033913/70795860-19eb1b80-1da1-11ea-8963-634decff0230.gif)

Reply:
Well, the diff for this PR shows that it is not instantiated anymore, so that's probably why:

![image](https://user-images.githubusercontent.com/591645/70796224-fd031800-1da1-11ea-89ca-627669571997.png)

Reply:
Thanks, @jtpio . Probably I removed that during rebasing